en/docs/security-announcements/cve-justifications/2026/CVE-2025-11143.md

```
@@ -0,0 +1,39 @@
+title: "CVE-2025-11143"
+category: security-announcements
+date: April 20 2026
+---
+
+# CVE-2025-11143
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)